### PR TITLE
PIA-1903: Rating feature implementation

### PR DESCRIPTION
**Pull request by kp-iva-nedeleva**

### CODE REVIEW

File: features/connection/src/main/java/com/kape/connection/ui/mobile/ConnectionScreen.kt [hunk 1]
```diff
@@ -171,6 +186,66 @@ fun ConnectionScreen() = Screen {
...
+                                launchIntent.data = Uri.parse(url)
+
+                                // Silently fail if Google Play Store isn't installed.
+                                if (launchIntent.resolveActivity(context.packageManager) != null) {
+                                    context.startActivity(launchIntent)
+                                }
+                                viewModel.setRatingStateInactive()
+                            },
+                            onDismiss = {
+                                viewModel.setRatingStateInactive()
```
Comment (kp-juan-docal) [inline features/connection/src/main/java/com/kape/connection/ui/mobile/ConnectionScreen.kt:224]: I haven't tried to debug it, but this doesn't seem to be working as expected. I can't dismiss the dialog.

Comment (kp-iva-nedeleva) [reply features/connection/src/main/java/com/kape/connection/ui/mobile/ConnectionScreen.kt:224]: Hmmm, I'll have a look. Thanks!

File: features/connection/src/main/java/com/kape/connection/ui/vm/ConnectionViewModel.kt [hunk 2]
```diff
@@ -330,8 +336,27 @@ class ConnectionViewModel(
...
+        _state.value = state.value.copy(ratingDialogType = RatingDialogType.Review)
+    }
+
+    fun showFeedbackPrompt() {
+        _state.value = state.value.copy(ratingDialogType = RatingDialogType.Feedback)
+    }
+
+    fun setRatingStateInactive() {
+        ratingTool.setRatingInactive()
+        _state.value = state.value.copy(ratingDialogType = null)
```
Comment (kp-iva-nedeleva) [inline features/connection/src/main/java/com/kape/connection/ui/vm/ConnectionViewModel.kt:355]: This line was missing, it now works as expected.

File: features/connection/src/main/java/com/kape/connection/ui/mobile/ConnectionScreen.kt (hunk 1 above)
Comment (kp-iva-nedeleva) [reply features/connection/src/main/java/com/kape/connection/ui/mobile/ConnectionScreen.kt:224]: Addressed.

**Review by kp-juan-docal — APPROVED** [review]

**Pull request merged**